app/controllers/concerns/dsi_authenticatable.rb

```diff
@@ -16,6 +19,12 @@ def authenticate_dsi_user!
     end
   end
 
+  def enforce_terms_and_conditions_acceptance!
+    if FeatureFlags::FeatureFlag.active?("terms_and_conditions") && current_dsi_user.terms_and_conditions_version_accepted != CURRENT_TERMS_AND_CONDITIONS_VERSION
+        redirect_to check_records_terms_and_conditions_path
+    end
```

Comment:
This check needs updating so we re-prompt them every six months (or whatever value was agreed).